repos/rust-lang/crates.io-index.toml

```
@@ -0,0 +1,12 @@
+org = "rust-lang"
+name = "crates.io-index"
+description = "Registry index for crates.io"
```

To accommodate #1353.

```suggestion
description = "Registry index for crates.io"
homepage = "https://crates.io"
```